src/api/handlers/action.py

```diff
@@ -113,15 +113,22 @@ def list(self, request):
 
     self.validator.expect('community_id', int, is_required=False)
     self.validator.expect('subdomain', str, is_required=False)
+    self.validator.expect('limit', int, is_required=False)
+
 
     args, err = self.validator.verify(args)
     if err:
       return err
 
     action_info, err = self.service.list_actions(context, args)
+
     if err:
       return err
-    return MassenergizeResponse(data=action_info)
+
+    meta = action_info.get('meta')
+    data = action_info.get('items')
+    return MassenergizeResponse(data=data, meta=meta)
```

Comment:
why dont you pass the data in and do this split in the MassenergizeREesponse itself?  then we will reduce a lot of the file changes even.